[Maps] Getting started docs - Adding the map to a dashboard (#35636)
elastic/kibana: docs/maps/maps-getting-started.asciidoc

===== Join the vector layer with the sample web log index

You now have a vector layer containing the world countries.

> In order to symbolize countries by web traffic, you will need to augment the world country features with the count of Elasticsearch weblog documents originating from each country.

Reviewer: ```suggestion
To symbolize countries by web traffic, you'll need to augment the world country features with the count of Elasticsearch weblog documents originating from each country.
```

You must add the web log traffic property to the world countries so
that the property is available for styling.
You'll create a <<terms-join, terms join>> to link the vector source *World Countries* to

> the {es} index `kibana_sample_data_logs` on the shared key iso2 = geo.src.

Reviewer: Let's add some wording to better explain the terms join.  See the wording in the video script.

===== Set the vector style

The final step is to set the vector fill color to shade

> the countries by web log traffic.

Reviewer: How about adding some wording here about the static vs. dynamic styling.

===== Set the vector style

All of the world countries are still a single color because the layer is using <<maps-vector-style-static, static styling>>.

> Lets use <<maps-vector-style-data-driven, data driven styling>> to shade the world countries based on which country is sending the most requests.

Reviewer: Suggest changing this because we haven't really used "we" in this tutorial.

To shade the word countries based on which country is sending the most requests, you'll need to use <<maps-vector-style-data-driven, data driven styling>>.

[[maps-add-elasticsearch-layer]]
=== Adding layers for {es} data

To avoid overwhelming the user with too much data at once, you'll add two layers for {es} data.

> . The first layer will show aggregated data that represents many documents.

Reviewer: Make this a bulleted list and reverse the order to match the order of the sections in the doc.

The layer will appear when the user zooms out the map to show larger amounts of the globe.
. The second layer will display individual documents.
The layer will appear when the user zooms in the map to show smaller regions.

==== Add a vector layer from the document source

This layer displays web log documents as points.
The layer is only visible when users zoom in the map past zoom level 9.

. In the map legend, click *Add layer*.
. Click the *Documents* data source.
. Set *Index pattern* to *kibana_sample_data_logs*.
. Click the *Add layer* button.
. Set *Layer name* to `Actual Requests`.
. Set *Zoom range for layer visibility* to the range [9, 24].
. Set *Layer transparency* to 1.
. Set *Fill color* to *#2200ff*.

> . Click *Save & close*.

Reviewer: Some suggestions:

- Include an an image of what the map looks like in this zoom range.
- Change the zoom range to 5-24

Author: I opted to include a screen shot of the app between zoom levels 9 and 24

. Set *Index pattern* to *kibana_sample_data_logs*.
. Click the *Add layer* button.
. Set *Layer name* to `Total Requests and Bytes`.

> . Set *Zoom range for layer visibility* to the range [0, 9].

Reviewer: Where you show the map at the end of the page, instead of "Your map now looks like this" say "Your map now looks like this between between zoom levels x and x'>

. In the application toolbar, click *Save*.
. Enter `Tutorial web logs map` for the title.
. Click *Confirm Save*.
+
You have completed the steps for re-creating the sample data map.

*Next steps:*

> * Continue with this tutorial and use your map in a Kibana Dashboard.

Reviewer: How about making "use your map in a Kibana dashboard" a link to the next section in the tutorial.

Note:  use a lower case d for dashboard.
